src/github.com/getlantern/peerscanner/dsp/dsp_test.go

```diff
@@ -39,6 +40,8 @@ func TestRegister(t *testing.T) {
 		assert.NotNil(t, rec, "A new record should have been returned")
 	}
 
+	// give dnssimple enough time to complete the operation
+	time.Sleep(100 * time.Millisecond)
```

Comment:
I see that you've tried with 100 millis and 1 sec. In order to make it resilient, could we somehow detect success on the DNSSimple side to continue the test? Or maybe don't consider it failed if the fail was on DNSSimple side?